e2e/test/DeviceStreamingE2ETests.cs

```diff
@@ -289,7 +294,7 @@ private async Task TestDeviceStreamingAsync(TestDeviceType type, ITransportSetti
 
                 Assert.IsNotNull(clientRequest, "Received an unexpected null device streaming request");
 
-                _log.WriteLine("Device streaming request received (name=" + clientRequest.Name + "; url=" + clientRequest.Url + "; authToken=" + clientRequest.AuthorizationToken + ")");
+                _log.WriteLine("Device streaming request received (name=" + clientRequest.Name + "; uri=" + clientRequest.Uri + "; authToken=" + clientRequest.AuthorizationToken + ")");
```

Comment:
If you are up for it, I'd change these to string interpolation to make them easier to read.

Reply:
It's a bit out of scope for this PR